lib/tdiary/plugin/00default.rb

```diff
@@ -331,14 +331,27 @@ def default_ogp
 		uri = @conf.index.dup
 		uri[0, 0] = base_url if %r|^https?://|i !~ @conf.index
 		uri.gsub!( %r|/\./|, '/' )
-		image = File.join(uri, "#{theme_url}/ogimage.png")
+		image = @conf.banner.empty? ? File.join(uri, "#{theme_url}/ogimage.png") : @conf.banner
+		ogp = {
+			'og:title' => title_tag.gsub(/<[^>]*>/, ""),
+			'og:image' => (h image),
+		}
+		ogp['fb:app_id'] = @conf['ogp.facebook.app_id']
+		ogp['fb:admins'] = @conf['ogp.facebook.admins']
```

Comment:
These two lines are for keeping [compatibe with ogp plugin](https://github.com/tdiary/tdiary-contrib/blob/master/plugin/ogp.rb#L32-L33).